app/src/main/java/com/infomaniak/mail/ui/main/user/AttachMailboxFragment.kt

```diff
@@ -50,14 +54,18 @@ class AttachMailboxFragment : Fragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?): Unit = with(binding) {
         super.onViewCreated(view, savedInstanceState)
 
-        mailInput.showKeyboard()
+        mailInput.apply {
+            showKeyboard()
+            manageErrorOnTextChange(mailInputLayout, attachMailboxButton)
+
+            onFocusChangeListener = OnFocusChangeListener { _, hasFocus -> manageErrorOnFocusChange(hasFocus) }
```

Comment:
Maybe add the info that manageErrorOnFocusChange is only for the email field in its name, something like `manageEmailErrorOnFocusChange`